specs/mediawiki/v1/content.yaml

```diff
@@ -1142,10 +1143,17 @@ paths:
           type: integer
           required: true
         - name: sections
-          in: formData
+          in: body
```

Comment:
@d00rman there's [a ticket](https://phabricator.wikimedia.org/T116336) to figure out which of these to leave. 

The current situation (when we accept both an object and form-data with a single field) is impossible to express with swagger specification.